tasktypes/commitalert.py

```diff
@@ -31,6 +31,16 @@ def process_task(self, library, task):
             return
 
         newest_commit = unseen_upstream_commits[-1]
+        existing_job = self.dbProvider.get_job(library, newest_commit.revision, limit_by_ff_version=False)
```

Comment:
Won't this always set `existing_job` to `None` (and is that what we want)?

Reply:
When this happens; there is no currently existing_job (the variable is first defined here.)

The result of get_job won't always be None though. because we set limit_by_ff_version=False, we may retrieve a job that was for this library+commit but a different FF version.